pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,190 @@
+---
+layout: 'guide'
+title: 'JavaScript'
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate JavaScript code targeted at the following platforms
+
+- Web
+- React Native
+- Node.js
+
+### Quickstart
+
+Avo functions usage consists of 3 steps.
+
+#### Step 1. Include the Avo file
+
+##### Avo CLI is available through npm
+
+To install it and get the generated Avo file run the following commands.
+
+```bash
+$ npm install -g avo
+$ avo login
+$ avo pull --branch my-branch-name
+```
+
+Learn more about <Link passHref href="/implementation/cli"><a>the CLI here</a></Link>.
+
+You can also <Link passHref href="/explore-tracking-plan/download-or-copy-avo-file-manually"><a>download the file manually</a></Link> from your Avo workspace.
+
+#### Step 2. Initialize Avo
+
+Import Avo from the generated file and initialize it by calling the `initAvo` method before tracking
+
+```javascript
+function initAvo(options, systemProperties, 
+                 destinationOptions, [customDestDestination])
+```
+
+The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/devs-101#avo-generated-file-init"><a>the initialization overview</a></Link>
+
+and you call it like this
+
+```javascript
+import Avo from './Avo';
+
+Avo.initAvo({ env: 'dev' }, { ...systemProperties }, {});
+```
```

Comment:
Now when we have the reference documented below I think we can have the code snippet above more example oriented and basically skip this part